packages/modules/src/modules/settings/parts/sidebar/SettingsSidebarSearchInput.vue

```diff
@@ -0,0 +1,103 @@
+<script setup lang="ts">
+import type { VcKeyboardShortcutProps } from '@wisemen/vue-core-components'
+import {
+  VcIconButton,
+  VcKeyboardShortcut,
+  VcKeyboardShortcutProvider,
+  VcTextFieldIconLeft,
+  VcTextFieldInput,
+  VcTextFieldRoot,
+} from '@wisemen/vue-core-components'
+import {
+  AnimatePresence,
+  Motion,
+} from 'motion-v'
+import { ListboxFilter } from 'reka-ui'
+import {
+  computed,
+  useId,
+} from 'vue'
+import { useI18n } from 'vue-i18n'
+
+import { useInjectSettingsContext } from '@/modules/settings/settings.context'
+
+const { searchTerm } = useInjectSettingsContext()
+
+const { t } = useI18n()
+const id = useId()
+
+const SHORTCUT_KEYS: VcKeyboardShortcutProps['keyboardKeys'] = [
+  'meta',
+  'f',
+]
+
+function onClearSearchTerm(): void {
+  searchTerm.value = ''
+}
+
+const hasSearchTerm = computed<boolean>(() => searchTerm.value.trim().length > 0)
+</script>
+
+<template>
+  <VcKeyboardShortcutProvider
+    :keyboard-keys="SHORTCUT_KEYS"
+    :prevent-default="true"
+  >
+    <label
+      :for="id"
+      class="sr-only"
+    >
+      {{ t('module.settings.search.placeholder') }}
```

Comment:
steken we de vertalingen mee in de vue-core of gaat elk project die zelf moeten definen? Mss logischer dat elk project die zelf moet hebben als je meerdere talen wilt ondersteunen 🤔